app/src/main/res/layout/rating_dialog.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.cardview.widget.CardView xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Use a material card anf why do you need progress bar ?

Reply:
To fetch the rating from the server. It will show the progress bar. And when the ratings is fetched it will be gone